content/docs/contributing.md

```diff
@@ -8,29 +8,37 @@ Participation in Open Source projects is highly encouraged, and the Open Source
 
 ### Summary
 
-- Upstream contributions are highly encouraged
+We highly encourage contributions to existing open source projects. There are some requirements before you start patching, and guides to help you figure out the right path:
+
+- Patches generally don't require any review from the Open Source Program Office
 - Double-check you are not sharing confidential information
-- Avoid contributions with information or code that gives us an advantage over competitors
-- Only sign whitelisted CLAs, reach out to the Open Source team for advice
+- Please associate your commit with your saucelabs.com email unless you have a history of contributing to the repo under a different email before your employment at Sauce Lavs. [How to associate your commit with a saucelabs.com email.](https://help.github.com/articles/setting-your-email-in-git/)
 
 ### Common Contribution Rules
+
+The following criteria must be met when contributing to an open source project:
+
+- Only contribute to public repositories on GitHub that are under the Apache 2, MIT, BSD*, LGPL*, GPL*, MPL, EPL, ISC, CC-BY, CC-BY-SA, OFL, MS-PL, Boost Software License, Unlicense, or Artistic licenses
```

Comment:
I think it is easier to say something like "contribute to GitHub (or GitLab?) repositories that are under the [license guide](https://github.com/saucelabs/saucelabs.github.io/blob/d21517deb63f9669f2c0f7d1578ea71b1911c0dc/content/docs/license-guide.md), we avoid duplicating info like this and it is more maintainable.